xml_schema/controller.xsd

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
the empty line is breaking controller addons schema validation (and the error is not upstream). Please make sure you also fix it in next iteration.

Reply:
Next iteration meaning ?

Reply:
I identified a few issues with current schemas in our tests (see logs in https://travis-ci.org/xbmc/addon-check/builds/409912337 and identified issues in https://github.com/xbmc/xbmc/pull/14237#issuecomment-409546715). When those are fixed we'll try again to see if everything is okay.